src/fmu/dataio/dataio.py

```diff
@@ -529,6 +527,13 @@ def _show_deprecations_or_notimplemented(self) -> None:
                 "Please remove it from the argument list.",
                 UserWarning,
             )
+        if self.table_include_index:
+            warn(
+                "The 'table_include_index' option is deprecated and has no effect. "
+                "To get the index included in your dataframe, reset the index"
```

Comment:
I think you need a blank space after 'index'

Reply:
:) yes, well spotted